src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -729,63 +552,60 @@ private boolean isReadFromNormal(final GATKRead rec) {
             vcb.filters(filters.isEmpty() ? VariantContext.PASSES_FILTERS : filters);
 
             if (printTCGAsampleHeader) {
-                final Genotype tumorGenotype = new GenotypeBuilder(originalVC.getGenotype(tumorSampleName)).name("TUMOR").make();
-                final Genotype normalGenotype = new GenotypeBuilder(originalVC.getGenotype(normalSampleName)).name("NORMAL").make();
+                final Genotype tumorGenotype = new GenotypeBuilder(originalVC.getGenotype(MTAC.tumorSampleName)).name("TUMOR").make();
+                final Genotype normalGenotype = new GenotypeBuilder(originalVC.getGenotype(MTAC.normalSampleName)).name("NORMAL").make();
                 vcb.genotypes(Arrays.asList(tumorGenotype, normalGenotype));
             }
             annotatedCalls.add(vcb.make());
         }
         return annotatedCalls;
     }
 
-    //TODO: See if we can share code with HaplotypeCaller's isActive
-    private class Mutect2AssemblyRegionEvaluator implements  AssemblyRegionEvaluator {
-        public ActivityProfileState isActive(final AlignmentContext context, final ReferenceContext ref, final FeatureContext featureContext) {
-            if( context == null || context.getBasePileup().isEmpty() ) {
-                return new ActivityProfileState(ref.getInterval(), 0.0);
-            }
+    public ActivityProfileState isActive(final AlignmentContext context, final ReferenceContext ref, final FeatureContext featureContext) {
```

Comment:
Yes we should.  Fixed in both classes.  There is never a reason to omit this annotation.